Classes/Command/ImportCommandController.php

```diff
@@ -298,7 +304,7 @@ protected function executeCommand(PresetPartDefinition $partSetting)
             $this->importService->persistEntities();
             return $count;
         } catch (\Exception $exception) {
-            $this->logger->logException($exception);
+            $this->throwableStorage->logThrowable($exception);
```

Comment:
Shouldn't it be:
```
$logMessage = $this->throwableStorage->logThrowable($exception);
$this->logger->error($logMessage, LogEnvironment::fromMethodName(__METHOD__));
```

Reply:
I replaced all logThrowable calls with your suggestion @dlubitz  Thank you.